package.json

```diff
@@ -10,7 +10,7 @@
     "classnames": "^2.2.5",
     "firebase": "^4.3.0",
     "history": "^4.7.2",
-    "hls.js": "^0.7.9",
+    "hls.js": "^0.8.2",
```

Comment:
This minor version is just a change to how the package is built - changing from browserify to webpack